Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/BaseTemplateBean.java

```diff
@@ -12,6 +12,7 @@
 package org.kitodo.data.database.beans;
 
 import java.util.Date;
+
```

Comment:
Why did you or your IDE change this file only for this missing white line separator between different packages?

Reply:
This time, the IDE can not help it. I did that because there were too many checkstyle warnings, so I eliminated some.